csharp/ql/lib/semmle/code/csharp/dispatch/RuntimeCallable.qll

```diff
@@ -15,7 +15,10 @@ private import dotnet
 class RuntimeCallable extends DotNet::Callable {
   RuntimeCallable() {
     not this.(Modifiable).isAbstract() and
-    not getDeclaringType() instanceof Interface
+    (
```

Comment:
Do you need to do another merge from `main` to get rid of this, now that https://github.com/github/codeql/pull/6630 has been merged?